laborator/content/reprezentare-numere/solution/5-odd_even/odd_even.c

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <stdio.h>
+#include <stdlib.h>
+
+void print_binary(int number, int nr_bits)
+{
+	int i;
+	char *bits;
+
+	bits = malloc(sizeof(char) * nr_bits);
```

Comment:
```suggestion
	bits = malloc(sizeof(*bits) * nr_bits);
```
It's recommended that you use `sizeof(variable)` instead of `sizeof(type)` so the size is updated automatically if you change the type of the variable.